### ISSUE TITLE
Understanding   Timeouts

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/wcag#349]
_From @awkawk on March 2, 2018 15:20_

<p>The <a   href="https://www.w3.org/TR/WCAG21/#timeouts"   target="_blank">2.2.6 Timeouts (AAA) SC</a> has draft   Understanding content that can be viewed at <a   href="https://rawgit.com/w3c/wcag21/timeouts/understanding/21/timeouts.html"   target="_blank">Understanding   Timeouts</a>.</p><p>Please review the Understanding   document and submit a comment on this issue for any changes that are needed   to the content.</p>

_Copied from original issue: w3c/wcag21#785_

**Comment by michael-n-cooper** [comment]
_From @lseeman on April 22, 2018 17:7_

suggest we rewrite"When there is a chance to extend a time limit, it should be noted that many users, within 20 seconds, cannot read instructions to extend a time limit.  Using simple text can reduce the time it takes to read these instructions."4
to
"When there is a chance to extend a time limit,  many users will require a few minites to read the instructions. When the time given to read the warning and instructions is short (such as 20 seconds) users with language or reading disabilities will not be able to extend the time limit.  Using simple text can reduce the time it takes to read these instructions.

The best way to conform

**Comment by michael-n-cooper** [comment]
_From @alastc on April 23, 2018 15:26_

Note comment from #830:
>I initially thought that 2.2.6 should be made a technique of 2.2.1. On closer reading, I see now the hole it's trying to fill. Please include something near the beginning of the Intent section of each SC to clarify the difference.

**Comment by fstrr** [comment]
This issue looks to be inactive, so I'm closing it. Please re-open if necessary.
